packages/actor-function-factory-not/README.md

```diff
@@ -0,0 +1,41 @@
+# Comunica Not Function Factory Actor
+
+[![npm version](https://badge.fury.io/js/%40comunica%2Factor-function-factory-not.svg)](https://www.npmjs.com/package/@comunica/actor-function-factory-not)
+
+A comunica Not Function Factory Actor.
+
+This module is part of the [Comunica framework](https://github.com/comunica/comunica),
+and should only be used by [developers that want to build their own query engine](https://comunica.dev/docs/modify/).
+
+[Click here if you just want to query with Comunica](https://comunica.dev/docs/query/).
+
+## Install
+
+```bash
+$ yarn add @comunica/actor-function-factory-not
+```
+
+## Configure
+
+After installing, this package can be added to your engine's configuration as follows:
+```text
+{
+  "@context": [
+    ...
+    "https://linkedsoftwaredependencies.org/bundles/npm/@comunica/actor-function-factory-not/^1.0.0/components/context.jsonld"  
+  ],
+  "actors": [
+    ...
+    {
+      "@id": "urn:comunica:default:function-factory/actors#not",
+      "@type": "ActorFunctionFactoryNot"
+    }
+  ]
+}
+```
+
+### Config Parameters
```

Comment:
The script can probably be changed to remove this section in the readme.

Reply:
Yes! Is good. Is that all? :D

Reply:
As far as I see atm, yes, this looks good.
I do think the linter will complain about some things still, but if those are fixed, this looks ok to me.